pywhat/Data/regex.json

```diff
@@ -83,6 +83,46 @@
           "Finance"
        ]
     },
+    {
+       "Name": "Monero (XMR) Wallet Address",
+       "Regex": "^4([0-9]|[A-B])[a-zA-Z0-9]{93}$",
+       "Description": null,
+       "Rarity": 0.5,
+       "Tags": [
+          "Cryptocurrency",
+          "Finance"
+       ]
+    },
+    {
+       "Name": "Litecoin (LTC) Wallet Address",
+       "Regex": "^(L|M)[a-zA-Z0-9]{33}$",
+       "Description": null,
+       "Rarity": 0.5,
+       "Tags": [
+          "Cryptocurrency",
+          "Finance"
+       ]
+    },
+    {
+       "Name": "Bitcoin Cash (BCH) Wallet Address",
+       "Regex": "^bitcoincash:[a-zA-Z0-9]{42}$",

```

Comment:
This is much higher as it says bitcoincash at the front, i'd give it a rarity of 1.0